src/main/java/com/basho/riak/client/core/query/timeseries/FullColumnDescription.java

```diff
@@ -0,0 +1,180 @@
+package com.basho.riak.client.core.query.timeseries;
+
+/**
+ * Holds a complete definition for a Table Column in Time Series Riak.
+ * Immutable once created.
+ *
+ * @author Alex Moore <amoore at basho dot com>
+ * @since 2.0.4
+ */
+public class FullColumnDescription
+{
+    private final ColumnDescription baseColumnDescription;
```

Comment:
I started out that way, but since we don't have any cross-use cases between the two I think it's better to keep them separate.  If we want to add inheritance later we can without breaking stuff.  If we ever wanted to remove inheritance though we'd be out of luck. :)